src/PostgREST/Types.hs

```diff
@@ -111,9 +110,7 @@ unUniformObjects (UniformObjects objs) = objs
 -- | When Hasql supports the COPY command then we can
 -- have a special payload just for CSV, but until
 -- then CSV is converted to a JSON array.
-data Payload = PayloadJSON UniformObjects
-             | PayloadParseError BS.ByteString
-             deriving (Show, Eq)
+data Payload = PayloadJSON UniformObjects deriving (Show, Eq)
```

Comment:
Is there a reason to keep Payload as a type now that there is only one constructor? Perhaps we could use UniformObjects throughout the code instead.